src/Interpreters/DatabaseCatalog.cpp

```diff
@@ -1459,11 +1467,35 @@ void DatabaseCatalog::dropTableFinally(const TableMarkedAsDropped & table)
         table.table->drop();
     }
 
+    /// Check if we are interested in a particular disk
+    ///   or it is better to bypass it e.g. to avoid interactions with a remote storage
+    auto look_on_disk =
```

Comment:
Why do we need this on DatabaseCatalog?

Reply:
I initially thought `DatabaseCatalog` was somewhat related to data lakes catalogs lol. It makes sense.